project_categ/models/project_categ_model.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# © 2013 Daniel Reis
+# © 2017 Rigoberto Martínez <rigo1985@gmail.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    task_tag_id = fields.Many2one('project.tags', string='Root Tag for Tasks')
```

Comment:
No, this is a helper field with the root of the allowed tags.
The field name is not as expressive as it could, but it's best to keep it stable from former versions.